tyk-control-plane/values.yaml

```diff
@@ -1224,7 +1224,10 @@ tyk-mdcb:
     # Users should use httpPort instead of probes.healthCheckPort for newer versions.
     # This value is used to set TYK_MDCB_HTTPPORT when MDCB is >= 2.6.0.
     httpPort: 8181
-
+    # settings for the health checker
+    healthcheck:
```

Comment:
Can you update liveliness and readiness probes in this file too?
Also settings for http_ingress is missing here.